ci/prow/deployments/boskos_volumes.yaml

```diff
@@ -1,34 +0,0 @@
-# Copyright 2019 The Knative Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# Boskos deployment for Knative Prow instance
-apiVersion: v1
-kind: PersistentVolume
```

Comment:
If you're removing this file, when creating a new cluster (or recreating boskos), won't it miss the configuration of the persistent volume used by boskos?

Reply:
Wait... So the new boskos doesn't need persistent storage anymore?

Reply:
yeah that's right, I have double checked with the author of the PR in k8s/test-infra, Boskos has stopped using persistent storage about a year ago, and we are already using it. So these are just like cleaning up

Reply:
Awesome. Let's delete that volume then once this is in. :)

/lgtm
/approve